wcfsetup/install/files/acp/update_5.4_removeFiles.php

```diff
@@ -0,0 +1,22 @@
+<?php
+use wcf\system\WCF;
+
+$files = [
+	'lib/system/database/table/DatabaseTableUtil.class.php',
+];
+
+$sql = "DELETE FROM     wcf".WCF_N."_package_installation_file_log
+	WHERE           packageID = ?
+	                AND filename = ?";
+$statement = WCF::getDB()->prepareStatement($sql);
+
+foreach ($files as $file) {
```

Comment:
Add a query that checks whether the files actually belong to WoltLab Suite Core before deleting anything. Better play safe.